src/pages/jeongwan/Login/Login.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import "./Login.scss";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+
+const Login = () => {
+  // 메인으로 가는 함수
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate("/jeongwan-main");
+  };
+
+  //사용자 입력 데이터 저장
+
+  const [id, setId] = useState("");
+  const [pw, setPw] = useState("");
+
+  const saveUserId = (event) => {
+    // console.log(event.target.value);
+    setId(event.target.value);
+    // console.log(event.target.value);
+  };
+
+  const saveUserPw = (event) => {
+    setPw(event.target.value);
+    // console.log(event.target.value);
+  };
+
+  //유효성 검사 변수
+  const validation = id.includes("@") && pw.length >= 5;
+
+  return (
+    <div className="login">
+      <div id="content_wrap">
+        <h1 className="logo_name">westagram</h1>
```

Comment:
className을 작성할 땐 camelCase로 작성해주세요!
```suggestion
      <div id="contentWrap">
        <h1 className="logoName">westagram</h1>
```